packages/client-sdk-nodejs/src/internal/internal-auth-client.ts

```diff
@@ -208,10 +211,22 @@ export class InternalAuthClient implements IAuthClient {
       return new GenerateDisposableToken.Error(normalizeSdkError(err as Error));
     }
 
+    const tokenID = disposableTokenProps?.tokenID;
```

Comment:
supernit: I think the idiomatic casing for this variable name in TS/JS would be `tokenId`. I don't really care about it here for a local var, but we want to be deliberate in the props object since that is effectively public API.